tests/failing.llvm

```diff
@@ -0,0 +1,312 @@
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stRefundTest/refund_CallToSuicideTwice.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stRefundTest/refund_multimpleSuicide.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stRefundTest/refund_singleSuicide.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stRefundTest/refund_TxToSuicide.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stRefundTest/refundFF.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stRefundTest/refundSuicide50procentCap.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stRevertTest/RevertInCreateInInit_Paris.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stRevertTest/RevertInCreateInInit.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stRevertTest/TouchToEmptyAccountRevert3_Paris.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stSelfBalance/diffPlaces.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stSolidityTest/SelfDestruct.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stSpecialTest/eoaEmpty.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stSpecialTest/eoaEmptyParis.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stSpecialTest/failed_tx_xcf416c53_Paris.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stSpecialTest/selfdestructEIP2929.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stSStoreTest/InitCollision.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stSStoreTest/InitCollisionParis.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stStaticCall/static_ABAcallsSuicide0.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stStaticCall/static_callcodecall_10_SuicideEnd.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stStaticCall/static_callcodecall_10_SuicideEnd2.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stStaticCall/static_callcodecallcodecall_110_SuicideEnd.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stStaticCall/static_callcodecallcodecall_110_SuicideEnd2.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stStaticCall/static_callcodecallcodecall_110_SuicideMiddle.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stStaticCall/static_callcodecallcodecall_110_SuicideMiddle2.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stStaticCall/static_refund_CallToSuicideTwice.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stSystemOperationsTest/ABAcallsSuicide0.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stSystemOperationsTest/doubleSelfdestructTest.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stSystemOperationsTest/doubleSelfdestructTouch_Paris.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stSystemOperationsTest/extcodecopy.json
```

Comment:
@anvacaru this is a lot of tests being added, which seems overkill. Can we just skip tests that have the Dencun schedule? Are all these tests new tests in the test-suite update?

At this point, I'm inclined to say we should instead be disabling tests a bit more fine-grained. This list could be: `file_name:test_name`, so that we only disabled the DenCun tests.

This is too many to disable at once.

Reply:
> Are all these tests new tests in the test-suite update?

It's a bit hard to tell since the version change is not an incremental bump.

>At this point, I'm inclined to say we should instead be disabling tests a bit more fine-grained. This list could be: file_name:test_name, so that we only disabled the DenCun tests.

I will open a new PR with the improved test handler.

Reply:
Non Dencun tests are failing as well because the test files contain new unsupported fields like `rlp_decoded` and [expectException](https://github.com/ethereum/tests/blob/7867ad622290f76a6bff1a41f76e3ec885e7ffe7/docs/blockchain-tutorial.rst?plain=1#L276). I'll skip the `rlp_decoded`, since values are parsed from decoding the `rlp` field, not sure how to handle `expectException`.